client/js/components/Container.jsx

```diff
@@ -17,13 +17,19 @@ export default class Container extends React.Component {
     }
 
     render() {
+        // Pass props to dynamically rendered children
+        var props = { slideUp: this.slideUp.bind(this) };
+        const children = React.Children.map(this.props.children, (element) =>
+            React.cloneElement(element, props)
+        );
```

Comment:
A few lines down, the children are rendered using `{this.props.children}`

I wasn't sure how to add other props into the children